src/main/scala/viper/carbon/modules/impls/DefaultFuncPredModule.scala

```diff
@@ -632,7 +631,7 @@ with DefinednessComponent with ExhaleComponent with InhaleComponent {
       MaybeCommentBlock("Translate function body",
       translateResult(res) := translateExp(f.body.get))
     val checkPost = checkFunctionPostconditionDefinedness(f)
-    val body : Stmt = Seq(init, initOld, checkPre, checkExp, exp, checkPost)
+    val body : Stmt = Seq(init, checkPre, checkExp, exp, checkPost)
```

Comment:
old expressions cannot appear in function specifications, so we do not need to initialize an old state